src/main/resources/templates/dev-hub/authentication.html

```diff
@@ -10,57 +10,48 @@
 <body>
 
 <div layout:fragment="content">
-  <h1 class="govuk-heading-xl">Authentication</h1>
+  <h1 class="govuk-heading-xl">How to add an API key</h1>
   <h2 class="govuk-heading-l">API key authentication</h2>
 
-  <p class="govuk-body">The Companies House API requires
-    authentication credentials, in the form of an API key, to be
-    sent with each request.</p>
-
-  <p class="govuk-body">
-    To obtain an API key, go to <a href=https://www.example.com
-                                   class="govuk-link">Your applications</a> and register
-    the application with the Companies House Developer Hub as an
-    <code>API Key</code>
-    application. This will allocate a unique key to the
-    application which can be sent with any
-    <code>GET</code>
-    request for a public resource served by the Companies House
-    API.
+  <p class="govuk-body">The Companies House API requires authentication credentials to be sent with
+    each request, which take the form of an API key.</p>
+  <p class="govuk-body">To add an API key:
   </p>
+  <ol class="list list-number govuk-body">
+    <li>Go to <a href="https://www.example.com/">Application overview</a></li>
```

Comment:
I think there is an Overview page, please link to it.

Reply:
I think this is referring to the logged in page given the context that it then goes on to create it.